blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/IBlueRun.java

```diff
@@ -0,0 +1,166 @@
+package io.jenkins.blueocean.rest.model;
+
+import io.jenkins.blueocean.rest.Navigable;
+import io.jenkins.blueocean.rest.annotation.Capability;
+import org.kohsuke.stapler.export.Exported;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+
+import static io.jenkins.blueocean.rest.model.KnownCapabilities.BLUE_RUN;
+
+/**
+ * Common interface for job run details
+ * <p>
+ * TODO: This isn't C#. Figure out what to call io.jenkins.blueocean.rest.model.BlueRun so we can call this BlueRun
```

Comment:
@sophistifunk Other option is to use `BlueRunItem` instead of `IBlueRun`. BlueRunItem is consistent with other `...Item` interfaces, such as `BlueRunnableItem`.